.gitignore

```diff
@@ -1 +1,2 @@
 /.build
+test*
```

Comment:
I don't think we should add this. It will ignore all files starting with 'test' in all subdirectories. Maybe we should rename the test files to have a .conf extension and ignore them a bit more specifically as: /test*.conf. This will be a more specific match of only these files in the root of the repo.

Or, perhaps it could be even better to add some example files: Example/test1.conf and Example/test2.conf, and refer to that location in the example code.

Reply:
Okay, I can do that. You're right, I didn't think about files in subdirectories having that name.

Reply:
I added some example files in this latest commit. I changed the file format to *.json since that is the format that is _expected_ by the library, to make it clear. Perhaps you disagree, but I think it makes sense. Also, these files make it easy to delete and modify them to see the behavior of the library.

Reply:
Looks good to me!